src/main/java/org/folio/entlinks/service/authority/PropagationAuthoritySourceFileService.java

```diff
@@ -0,0 +1,134 @@
+
+import java.util.UUID;
+import java.util.function.BiConsumer;
+import lombok.extern.log4j.Log4j2;
+import org.folio.entlinks.controller.converter.AuthoritySourceFileMapper;
+import org.folio.entlinks.domain.entity.AuthoritySourceFile;
+import org.folio.entlinks.domain.repository.AuthorityRepository;
+import org.folio.entlinks.domain.repository.AuthoritySourceFileRepository;
+import org.folio.entlinks.exception.AuthoritySourceFileNotFoundException;
+import org.folio.entlinks.exception.OptimisticLockingException;
+import org.folio.entlinks.utils.JdbcUtils;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.FolioModuleMetadata;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Propagation;
+import org.springframework.transaction.annotation.Transactional;
+
+
+@Service("propagationAuthoritySourceFileService")
+@Log4j2
+public class PropagationAuthoritySourceFileService extends AuthoritySourceFileService {
+
+  private final JdbcTemplate jdbcTemplate;
+  private final AuthoritySourceFileRepository repository;
+  private final FolioExecutionContext folioExecutionContext;
+
+  public PropagationAuthoritySourceFileService(AuthoritySourceFileRepository repository,
+                                               AuthorityRepository authorityRepository,
+                                               AuthoritySourceFileMapper mapper,
+                                               JdbcTemplate jdbcTemplate,
+                                               FolioModuleMetadata moduleMetadata,
+                                               FolioExecutionContext folioExecutionContext) {
+    super(repository, authorityRepository, mapper, jdbcTemplate, moduleMetadata, folioExecutionContext);
+    this.jdbcTemplate = jdbcTemplate;
+    this.repository = repository;
+    this.folioExecutionContext = folioExecutionContext;
+  }
+
+  /**
+   * Creates shadow copy of Authority Source File in the member tenant's database schema.
+   *
+   * @param entity Authority Source File created in the consortium central tenant
+   * @return Authority Source File created in the member tenant.
+   *
+   *   Note: As this method creates a shadow copy of already created authority source file, it assumes no
+   *   validation/initialization is required and the entity passed to the method had this validation/initialization.
+   */
+  @Override
+  @Transactional
+  public AuthoritySourceFile create(AuthoritySourceFile entity) {
+    log.debug("create:: Attempting to create AuthoritySourceFile [entity: {}]", entity);
+
+    var sourceType = getFullPath(folioExecutionContext, "authority_source_file_source");
+    var sqlValues = JdbcUtils.getParamPlaceholder(3)
+        + "::" + sourceType + ","
+        + JdbcUtils.getParamPlaceholder(9);
```

Comment:
Would it be better to use String.format() here?
String sqlValues = String.format("%s::%s,%s", 
    JdbcUtils.getParamPlaceholder(3), 
    sourceType, 
    JdbcUtils.getParamPlaceholder(9)
);